server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java

```diff
@@ -204,12 +208,183 @@ public void runAddAnyMembershipTypeTransaction_owner() throws Exception {
   }
 
   @Test
-  public void getDeleteMembershipTransaction() throws Exception {
+  public void runDeleteMembershipTransaction_member() throws Exception {
     StorageHandlerTestHelper.insertPerson("person");
     StorageHandlerTestHelper.insertMembership("person", "club", MembershipConstants.MEMBER);
     StorageHandler.runDeleteMembershipTransaction(dbClient, "person", "club");
     ReadContext readContext = dbClient.singleUse();
     Boolean actual = StorageHandlerHelper.checkAnyMembership(readContext, "person", "club");
     assertFalse(actual);
   }
+
+  @Test
+  public void runDeleteMembershipTransaction_ownerFailsToLeaveClub() throws Exception {
+    StorageHandlerTestHelper.insertPerson("person");
+    StorageHandlerTestHelper.insertMembership("person", "club", MembershipConstants.OWNER);
+    ReadContext readContext = dbClient.singleUse();
+    assertThrows(RuntimeException.class, () -> {
+      StorageHandler.runDeleteMembershipTransaction(dbClient, "person", "club");
+    });
+  }
+
+  @Test
+  public void getListOfMembers_clubDoesNotExistInDb() throws Exception {
+    assertThrows(RuntimeException.class, () -> {
+      StorageHandler.getPerson(dbClient, "club");
+    });
+  }
+
+  @Test
+  public void getListOfMembers_none() throws Exception {
+    StorageHandlerTestHelper.insertClub("club", /* owner_id= */"owner");
+    assertThrows(RuntimeException.class, () -> {
+      StorageHandler.getListOfMembers(dbClient, "club");
+    });
+  }
+
+  @Test
+  public void getListOfMembers_oneOwner() throws Exception {
+    List<Person> expected = new ArrayList<Person>();
+    expected.add(StorageHandlerTestHelper.createTestPersonObject("owner", /* pronouns= */true));
+
+    StorageHandlerTestHelper.insertPerson("owner");
+    StorageHandlerTestHelper.insertClub("club", /* owner_id= */"owner");
+    StorageHandlerTestHelper.insertMembership("owner", "club", MembershipConstants.OWNER);
+    List<Person> actual =  new ArrayList<Person>(StorageHandler.getListOfMembers(dbClient, "club"));
```

Comment:
```suggestion
    List<Person> actual = new ArrayList<Person>(StorageHandler.getListOfMembers(dbClient, "club"));
```
Extra space, this occurs throughout the file.